src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
```

Comment:
There should not be a redundant empty space between the class declaration `extends` keyword and the parent class `Machine`. It should be `Bulldozer extends Machine`.